spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -1300,9 +1364,9 @@
 
     context "when county is Anne Arundel" do
       let(:county) { "Anne Arundel" }
-      it "uses 0.0270 as the local tax rate in the formula" do
-        # (0.027 * 10) * 1001
-        expect(instance.lines[:MD502_LINE_29].value).to eq(270)
+      it "uses 0.0281 as the local tax rate in the formula" do
+        # (0.0281 * 10) * 1001
+        expect(instance.lines[:MD502_LINE_29].value).to eq(281)
```

Comment:
should we flag to acceptance testers that the `EarnedIncomeCredit` value utilizes the highest local rate for Anne Arundel county now? [For the original story it seemed like we were saying to use 0.270 ](https://codeforamerica.atlassian.net/browse/FYST-770)

[Added a jira question here](https://codeforamerica.atlassian.net/browse/FYST-1453?focusedCommentId=14143)

Reply:
@arinchoi03 yes thank you for flagging!